src/routes/jira/jira-atlassian-connect-get.ts

```diff
@@ -90,6 +90,15 @@ const modules = {
 			url: "/jira/app-creation",
 			location: "none",
 			conditions: adminCondition
+		},
+		{
+			key: "github-manual-app-creation-page",
+			name: {
+				value: "GitHub Manual App Creation"
+			},
+			url: "/jira/manual-app-creation?ghsaId={ac.ghsaId}",
```

Comment:
if we're creating the app, why is there an id already generated for it?

Reply:
This page is for both creating and editing. Updating the name to just `manual-app`.